File: app/src/main/java/com/antyzero/smoksmog/ui/BaseFragment.java
package com.antyzero.smoksmog.ui;

import android.os.Bundle;
import android.support.annotation.CallSuper;
import android.view.View;

import com.trello.rxlifecycle.components.RxFragment;

import butterknife.ButterKnife;

/**
 * Base for all fragments
 */
public abstract class BaseFragment extends RxFragment {

    @Override
    @CallSuper
    public void onViewCreated( View view, Bundle savedInstanceState ) {
        super.onViewCreated( view, savedInstanceState );
        ButterKnife.bind( this, view );
    }
}

File: app/src/main/kotlin/com/antyzero/smoksmog/ui/BaseFragment.kt
package com.antyzero.smoksmog.ui

import android.os.Bundle
import android.support.annotation.CallSuper
import android.view.View
import com.trello.rxlifecycle.components.RxFragment

abstract class BaseFragment : RxFragment() {

    @CallSuper
    override fun onViewCreated(view: View?, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)
    }
}
